src/pkg/retention/policy/rule/latestpl/evaluator.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package lastpulled
+package latestpl
```

Comment:
The new package name feels more confusing to me. Was the old one too long?

Reply:
yes, too long